app/assets/stylesheets/components/_footer.scss

```diff
@@ -4,6 +4,9 @@
   margin-bottom: 30px;
   margin-top: 60px;
   text-align: center;
+  p {
```

Comment:
I think we should address `max-width` holistically, similar to how [Tailwind's `prose`](https://tailwindcss.com/docs/typography-plugin) handles maximum reading length.

We should address max-width handling on the container, rather than the lines of text themselves. See `_page.scss`:

```scss
.Page__content {
  grid-area: content;
  width: 100%;
  max-width: map-get($max-w, "prose");
  margin-inline: auto;
  padding-block: 24px;

  @media (min-width: $screen-lg) {
    justify-self: center;
    padding-block: 40px;
    &:nth-child(2) {
      max-width: 1074px;
    }

    h1,
    h2,
    h3,
    h4,
    h5,
    h6,
    p,
    ul {
      max-width: map-get($max-w, "prose");
    }
  }
}
```

So, rather than assigning max-width on h1...6, p and ul, we should set it on a container so it can cover other content elements, such as ol, table, dl/dt/dd et al.

I suggest we can use `.Article` as the `prose` equivalent.

Also, some nitpicking:
- We should avoid element selectors on a component level
- Need a line between CSS declarations (should really add a scss lint rule 🤔 )

Reply:
Turns out `max-width: map-get($max-w, "prose");` is already in place on the containing `.Page__content` element. Looks like this issue can be resolved simply by removing the rules under the media query as suggested.